install.php

```diff
@@ -207,6 +250,186 @@ public function installOrUpdate($parent)
 	}
 
 	/**
+	 * Method to process SQL updates previous to the install process
+	 *
+	 * @param   object  $parent  Class calling this method
+	 *
+	 * @return  boolean          True on success
+	 */
+	public function preprocessUpdates($parent)
```

Comment:
it will run twice in fact
Joomla is using its own when doing the schema updates, since it's an static function it doesn't matter if I try to override it because it can be overriden
so I'm separating it into a pre-processor to create the procedures  and then the regular Joomla schema upgrade to run regular queries and run the procedures